File: C++/Daily Flash/Week2/Day6_Jan26/code/prog2.cpp

#include <iostream>

int main() {
	int num1,num2;
	std::cout << "Enter num1: ";
	std::cin >> num1;
	
	std::cout << "Enter num2: ";
	std::cin >> num2;
        
        std::cout << "Before swapping\n";
	std::cout << num1 << "  " << num2 << std::endl;	
	int temp= num1;
	num1 = num2;
	num2 = temp;

	std::cout << "Before swapping\n";
	std::cout << num1 << "  " << num2 << std::endl;	
        return 0;
}

File: JAVA/Daily Flash Codes/Week2/Day6_Jan25/code/prog2.java


import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
		int num1,num2;
		System.out.println("Enter range to print odd num: ");
		System.out.println("Enter num1: ");
		num1 = 	Integer.parseInt(in.readLine());
		
		System.out.println("Enter num2: ");
		num2 = 	Integer.parseInt(in.readLine());
		
		System.out.println("before swap");	
		System.out.println(num1 + "  "+ num2);
			
		int temp = num1;
		num1= num2;
		num2 = temp;
		System.out.println("After swap");
		System.out.println(num1 + "  "+ num2);
	 
	}


}
